packages/azure_functions/data_stream/functionapplogs/manifest.yml

```
@@ -0,0 +1,59 @@
+title: Collect Azure Functions logs
+type: logs
+streams:
+  - input: "azure-eventhub"
+    enabled: false
+    template_path: "azure-eventhub.yml.hbs"
+    title: Azure Functions logs
+    description: Collect Azure Functions logs using azure-eventhub input
+    vars:
+      - name: preserve_original_event
+        required: true
+        show_user: true
+        title: Preserve original event
+        description: Preserves a raw copy of the original event, added to the field `event.original`
+        type: bool
+        multi: false
+        default: false
+      - name: storage_account_container
+        type: text
+        title: Storage Account Container
```

```suggestion
        title: Storage account container
```

Making it sentence case like all other fields